src/aap_eda/services/activation/tee_log_handler.py

```diff
@@ -0,0 +1,55 @@
+#  Copyright 2023 Red Hat, Inc.
+#
+#  Licensed under the Apache License, Version 2.0 (the "License");
+#  you may not use this file except in compliance with the License.
+#  You may obtain a copy of the License at
+#
+#      http://www.apache.org/licenses/LICENSE-2.0
+#
+#  Unless required by applicable law or agreed to in writing, software
+#  distributed under the License is distributed on an "AS IS" BASIS,
+#  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#  See the License for the specific language governing permissions and
+#  limitations under the License.
+
+import logging
+from typing import Union
+
+from aap_eda.services.activation.db_log_handler import DBLogger
+
+LOGGER = logging.getLogger(__name__)
+
+
+class TeeLogger(DBLogger):
+    def __init__(self, activation_instance_id: int):
```

Comment:
If you don't need to override or extend the initialization of the object there is no need to define `__init__` here

BUT, using the LOGGER of this module probably is wrong since all the records would be tracked by this logger and we would be losing the track of the real module that is calling it. 

Instead I think it is better to pass the logger in the constructor in a more dependency injector pattern and allow to be used by different implementers, We consume the DBLogger from the engines and from the manager but this new logger I guess is intent to be called only from the engine. 

thoughts @hsong-rh @ShaiahWren

Reply:
I think you are right @Alex-Izquierdo . I'm updating the pr now, and then I'll work on the test.

Reply:
Are you going to implement this new logger in this PR or in a separate PR?

Reply:
I will implement it in this pr, if that's alright with you. I also need to write the test this afternoon.